controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -632,54 +493,19 @@ public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenChec
     public CompletableFuture<Boolean> createTableSegment(final String tableName,
                                                          String delegationToken,
                                                          final long clientRequestId) {
-        final CompletableFuture<Boolean> result = new CompletableFuture<>();
```

Comment:
Observing `io.pravega.controller.server.SegmentHelper#createTransaction` still uses `sendRequestAsync` to send the request instead of RawClient. Any specific reason why we are not using the RawClient here?

Reply:
this changes are only for functions that return CompletableFuture<Boolean>; the reason to narrow down is to review less code to speed up merge (thank you all); after this part get merged, all the other function will be modified together in another PR.